src/v2/styles/ChannelSearch/ChannelSearch-theme.scss

```diff
@@ -0,0 +1,192 @@
+@use '../utils';
+
+.str-chat {
+  /* The text/icon color of the component */
+  --str-chat__channel-search-input-color: var(--str-chat__text-color);
+
+  /* The text/icon color of the component */
+  --str-chat__channel-search-input-placeholder-color: var(--str-chat__text-low-emphasis-color);
+
+  /* The text/icon color of the component */
+  --str-chat__channel-search-input-icon-color: var(--str-chat__text-low-emphasis-color);
+
+  /* Top border of the component */
+  --str-chat__channel-search-input-border-block-start: none;
+
+  /* Bottom border of the component */
+  --str-chat__channel-search-input-border-block-end: none;
+
+  /* Left (right in RTL layout) borhe component */
+  --str-chat__channel-search-input-border-inline-start: none;
+
+  /* Right (left in RTL layout) borhe component */
+  --str-chat__channel-search-input-border-inline-end: none;
+
+
+  /* The border radius used for the borders of the component */
+  --str-chat__channel-search-input-wrapper-border-radius: 20px;
+
+  /* The background color of the component */
+  --str-chat__channel-search-input-wrapper-background-color: transparent;
+
+  /* Top border of the component */
+  --str-chat__channel-search-input-wrapper-border-block-start: var(--str-chat__surface-color) 1px solid;
+
+  /* Bottom border of the component */
+  --str-chat__channel-search-input-wrapper-border-block-end: var(--str-chat__surface-color) 1px solid;
+
+  /* Left (right in RTL layout) border of the component */
+  --str-chat__channel-search-input-wrapper-border-inline-start: var(--str-chat__surface-color) 1px solid;
+
+  /* Right (left in RTL layout) border of the component */
+  --str-chat__channel-search-input-wrapper-border-inline-end: var(--str-chat__surface-color) 1px solid;
+
+  /* The border radius used for the borders of the component */
+  --str-chat__channel-search-input-wrapper-active-border-radius: 20px;
```

Comment:
I think it's better to use a theme variable for border-radius so integrators can easily customize it, we have `--str-chat__border-radius-md` which is 18px, if that's acceptable, or we can create a new one like `--str-chat__border-radius-lg` for 20px

Reply:
Thank you @szuperaz :+1: I have added a variable with the suffix `-lg`